helm/repo/index.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v1
 entries:
```

Comment:
Should we modify the existing index or create a new one?

Reply:
So far, we always use this one.